techlibs/gatemate/synth_gatemate.cc

```diff
@@ -303,11 +311,22 @@ struct SynthGateMatePass : public ScriptPass
 
 		if (check_label("map_luts"))
 		{
-			std::string abc_args = " -dress -lut 4";
-			if (dff) {
-				abc_args += " -dff";
+			if (luttree || help_mode) {
+				std::string abc_args = " -genlib +/gatemate/lut_tree_cells.genlib";
+				if (dff) {
+					abc_args += " -dff";
+				}
+				run("abc " + abc_args, "(with -luttree)");
+				run("techmap -map +/gatemate/lut_tree_map.v", "(with -luttree)");
+				run("gatemate_foldinv", "(with -luttree)");
+				run("techmap -map +/gatemate/inv_map.v", "(with -luttree)");
+			} else {
+				std::string abc_args = " -dress -lut 4";
+				if (dff) {
+					abc_args += " -dff";
+				}
+				run("abc " + abc_args, "(without -luttree)");

```

Comment:
The comment here never actually gets printed, since the if branch triggers in help mode, right?

Reply:
Good spot; should be fixed now to print both branches in help mode